examples/seismic/tutorials/15_tti_qp_pure.ipynb

```diff
@@ -0,0 +1,2933 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        psave[step,:,:]=p.data[(step+1)%3,:,:]
</pre><p>Maybe switch to <code>info</code>  or <code>error</code>  for the log here, quite the output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1752/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>